README.rst

```diff
@@ -40,34 +40,41 @@ You can install "pytest-splunk-addon" via `pip`_ from `PyPI`_::
 Developing
 ------------
 
-Note: Must install docker desktop.
+Note: Must setup kubernetes cluster [ Reference: `minikube`_ , `microk8s`_, `k3s`_ ], vscode or pycharm pro optional
+
+Note2: Appinspect requires libmagic verify this has been installed correctly each time a new workstation/vm is used https://dev.splunk.com/enterprise/docs/releaseapps/appinspect/splunkappinspectclitool/installappinspect
 
 .. code:: bash
 
     $ git clone --recurse-submodules -j8 git@github.com:splunk/pytest-splunk-addon.git
-    $ cd pytest-splunk-addon
-    $ poetry install
-    $ ... (change something)
-    # run unit tests
-    $ poetry run pytest tests/unit
-    # run some of the docker-based tests to verify end-to-end behaviour, example:
-    $ poetry run pytest -v --splunk-version=8.2 -m docker tests/test_splunk_addon.py::test_splunk_app_requirements_modinput
+    $ #setup python venv must be 3.7    
+    $ python3 -m venv .venv
 
+    $ source .venv/bin/activate
 
-Usage
------
+    $ cd pytest-splunk-addon
 
-Installation for external Splunk
+Generate addons spl (tests/e2e/addons/<ADDON_NAME>) with format ``<ADDON_NAME>-<ADDON_VERSION>.spl``
 
 .. code:: bash
 
-    pip install pytest-splunk-addon
+    $ pip install poetry
+
+    $ poetry export --without-hashes --dev -o requirements_dev.txt
+
+    $ pip install -r requirements_dev.txt
 
-Installation with built in docker orchestration
+    $ export KUBECONFIG="PATH of Kubernetes Config File"
```

Comment:
Can you please be more specific what should I put here?

Reply:
KUBECONFIG file path will vary for different kubernetes infrastructure setups, I have provided the examples for reference.